z3/src/sort.rs

```diff
@@ -154,6 +154,22 @@ impl<'ctx> Sort<'ctx> {
         unsafe { Z3_get_sort_kind(self.ctx.z3_ctx, self.z3_sort) }
     }
 
+    pub fn get_float_exponent_size(&self) -> Option<u32> {
```

Comment:
Please remove the `get_` prefix, as it is generally unidiomatic Rust (this crate isn't fully consistent with this practice, but gating new changes on being idiomatic helps move us in the right direction).